staff/aiida-montero/Challenges/calculator-app-grid/calculate.js

```diff
@@ -0,0 +1,110 @@
+
+var current = ''
```

Comment:
put semicolon in all lines that you forgot to set it

Reply:
Listo ;)